_posts/12-06-2022-Bazel-release-6.0-blog.md

```diff
@@ -0,0 +1,38 @@
+---
+layout: posts
+title: Bazel 6.0 LT Release blog post
+authors:
+  - radvani
+---
+
+# Bazel 6.0 launched!
+
+Today, we launch Bazel 6.0 LTS!. Bazel 6.0 delivers many requested customer updates to simplify Bazel usage and eliminate common Android app development roadblocks.
+
+![Image](../images/Bazel6.0ReleaseBlogPost.png)
+
+### Bzlmod becomes Generally available
+[Bzlmod](https://bazel.build/docs/bzlmod) automatically resolves transitive dependencies, allowing projects to scale while staying fast and resource-efficient. In development since 2021, Bzlmod enters general availability with Bazel 6.0.
+
+-   Pipelines to automatically check common dependencies and rulesets into the Bazel Central Registry.
+-   Updated [Bzlmod Migration Guide](https://docs.google.com/document/d/1JtXIVnXyFZ4bmbiBCr5gsTH4-opZAFf5DMMb-54kES0/edit?usp=gmail) includes scripts and documentation for migration.
+-   Support for `rules_jvm_external`, allowing users to download Maven dependencies for Java projects.
```

Comment:
This line reads like rules_jvm_external is new. How about "Added Bzlmod support for..."?